src/renderer/components/SettingsPage.tsx

```diff
@@ -801,6 +820,27 @@ export default class SettingsPage extends React.PureComponent<Record<string, nev
                 <FormText>
                     {'Specify the folder where files will download.'}
                 </FormText>
+                <br/>
+                {'Logging level'}
+                <FormControl
+                    style={settingsPage.logLevelInput}
+                    as='select'
+                    id='inputLogLevel'
+                    ref={this.logLevelRef}
+                    value={this.state.logLevel}
+                    onChange={this.handleChangeLogLevel}
+                >
+                    <option value='error'>{'Errors only (error)'}</option>
```

Comment:
Errors (error)
Errors and Warnings (warn)
Info (info)
Verbose (verbose)
Debug (debug)
Finest (silly)

Logging is helpful for developers and support to isolate issues you may be encountering with the desktop app. Increasing the log level increases disk space usage and can impact performance. We recommend only increasing the log level if you are having issues.

Reply:
@devinbinnie thoughts? ^

Reply:
Works for me, will make the changes.